modules/gitops-configure-rollout-route-traffic-using-openshift-service-mesh.adoc

```diff
@@ -60,11 +62,11 @@ spec:
     targetPort: http
     protocol: TCP
     name: http
-  Selector: #<2>
+  selector: #<2>
     app: rollouts-demo
 ----
 <1> Specifies the name of the port used by the application for running inside the container.
-<2> Ensure that the contents of the `selector` field are the same as in stable service and `Rollout` custom resource (CR).
+<2> Ensure that the contents of the selector field are the same in stable service and Rollout custom resource (CR).
```

Comment:
Not sure why this was requested, but `Rollout` is correct, not Rollout, AFAIK

Reply:
same with `selector`

Reply:
Fixed.